src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -212,10 +215,11 @@ public int allowedSpaceCount()
     }
 
     @NotNull
-    private List<String> getColonyTabCompletions(final String potentialArgumentValue)
+    private List<String> getColonyTabCompletions(@NotNull final MineColonyDataProvider mineColonyDataProvider,
+            final String potentialArgumentValue)
     {
         // TODO: use the colony we are in as the default tab completion.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)